onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -714,6 +714,36 @@ TEST_F(GraphTransformationTests, FuseCudaConvAddRelu) {
   ASSERT_TRUE(op_to_count["Relu"] == 0);  // Relu removed from graph
 }
 
+// Currently the ConvAddRelu fusion is only backed by a float kernel for the
+// the CUDA EP.
+
+// When we see the corresponding pattern for the fp16 data type, the fusion
+// should not be triggered as there is no kernel to back the fused pattern.
+
+// TODO(hasesh): Limit the test to using the CUDA EP for now as the level of
+// data type support in other compatible EPs is still yet to be ascertained.
+
+// TODO(hasesh): If it all the fp16 type is supported for the fusion, adjust/remove
```

Comment:
not quite following the comment
"If it all the fp16 type is supported for the fusion"

Reply:
That should read - "If at all,..." - does the comment make sense after that fix ?